aspnet/security/data-protection/consumer-apis/password-hashing.rst

```diff
@@ -1,16 +1,15 @@
 Password Hashing
```

Comment:
Did you mean to include these changes in this PR? They aren't lexer related, but they look ok.